File: 프로그래머스/Level1/K번째수.py
# https://programmers.co.kr/learn/courses/30/lessons/42748

def solution(array, commands):
    answer = []
    for i, j, k in commands:
        temp_arr = array[i-1:j]
        temp_arr.sort()
        answer.append(temp_arr[k-1])
    return answer

File: 프로그래머스/Level1/K번째 수.js
// https://programmers.co.kr/learn/courses/30/lessons/42748

const solution = (array, commands) => {
  const answer = [];
  commands.forEach((command) => {
    const [i, j, k] = command;
    const tempArray = array.slice(i - 1, j);
    tempArray.sort((a, b) => a - b);
    answer.push(tempArray[k - 1]);
  });
  return answer;
};
